GliaWidgets/SecureConversations/SecureConversations.Availability.swift

```diff
@@ -85,6 +98,15 @@ extension SecureConversations.Availability {
                 return false
             }
         }
+
+        var isAvailable: Bool {
+            switch self {
```

Comment:
Just a matter of readability for me, `self == .available` looks a bit straightforward.

Reply:
@ykyivskyi-gl  It has associated values, so compiler needs honest `switch` there, `self == .available` will not compile.

Reply:
Hm, indeed